climada/hazard/trop_cyclone.py

```diff
@@ -222,6 +229,34 @@ def from_tracks(
             "H08" (Holland 1980 with b-value from Holland 2008), "H10" (Holland et al. 2010), or
             "ER11" (Emanuel and Rotunno 2011).
             Default: "H08".
+        model_kwargs : dict, optional
+            If given, forward these kwargs to the selected wind model. None of the parameters is
+            currently supported by the ER11 model. The Holland models support the following
+            parameters, in alphabetical order:
+
+            gradient_to_surface_winds : float, optional
+                The gradient-to-surface wind reduction factor to use. In H1980, the wind profile is
+                computed on the gradient level, and wind speeds are converted to the surface level
+                using this factor. In H08 and H10, the wind profile is computed on the surface
+                level, but the clipping interval of the B-value depends on this factor.
+                Default: 0.9
+            rho_air_const : float or None, optional
+                The constant value for air density (in kg/m³) to assume in the formulas from
+                Holland 1980. By default, the constant value suggested in Holland 1980 is used. If
+                set to None, the air density is computed from pressure following equation (9) in
+                Holland et al. 2010. Default: 1.15
+            vmax_from_cen : boolean, optional
+                Only used in H10. If True, replace the recorded value of vmax along the track by
+                an estimate from pressure, following equation (8) in Holland et al. 2010.
+                Default: True
+            vmax_in_brackets : bool, optional
+                Only used in H10. Specifies which of the two formulas in equation (6) of Holland et
+                al. 2010 to use. If False, the formula with vmax outside of the brackets is used.
+                Note that, a side-effect of the formula with vmax inside of the brackets is that
+                the wind speed maximum is attained a bit farther away from the center than
+                according to the recorded radius of maximum winds (RMW). Default: False
```

Comment:
For me it is very unclear from the paper if `vmax_from_cen` and `vmax_in_brackets` "has to be combined". I agree they would often be used in the same combination, but we should give the user the freedom to choose themselves. For me it is very hard to describe the effect of the parameter change, because I imagine it is very dependent on the TC event and on other chosen settings... Maybe we can sit together and think about a clarification of this docstring. Otherwise I would leave it as suggest by @tovogt.

Reply:
Alright, let's leave it as is.